tests/test_gglobals.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from unittest.mock import Mock, patch
+
+from gourmand.gglobals import _get_link_and_star_color
+
+
+class TestLinkColor(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    @patch("gourmand.gglobals.Gtk")
+    def test_dark_mode_link_color(self, Gtk):
+        # Closer to 0 is darker.  So in dark mode the background
+        # will be a smaller value than the foreground color.
+        dark_bgcolor = Mock()
+        fgcolor = Mock()
+        Gtk.TextView().get_style_context().get_background_color.return_value = dark_bgcolor
+        Gtk.TextView().get_style_context().get_color.return_value = fgcolor
+        fgcolor.red = 1
+        fgcolor.green = 1
+        fgcolor.blue = 1
+        dark_bgcolor.red = 0.18
+        dark_bgcolor.green = 0.18
+        dark_bgcolor.blue = 0.19
+        link_color_actual, star_color_actual = \
```

Comment:
These most likely can go into one line.

Reply:
It seems like this has been a misunderstanding - I meant the current line 22 where you call the actual function, not the assignments.